src/components/phase-banner/index.md

```diff
@@ -23,6 +24,8 @@ Use an alpha banner when your service is in alpha, and a beta banner if your ser
 
 Your banner must be directly under the black GOV.UK header and colour bar.
 
+<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Do not make the phase banner ‘sticky’ to the top of the page by using `position: fixed` or any other method. This is to make sure it does not cover or obscure any content which has a focus applied and to comply with WCAG 2.2 success criterion 2.4.11 Focus not Obscured (minimum).
```

Comment:
@calvin-lau-sig7 this is what I'm seeing for the `WCAG 2.2` tag:
<img width="744" alt="screenshot of new WCAG 2.2 guidance on the phase banner component page, showing the WCAG 2.2 with a light grey background and a code snippet with the same light grey background" src="https://github.com/alphagov/govuk-design-system/assets/98318778/df3cb7cd-2f78-4e3d-a8a6-27230aa799f8">

Note that the code snippet and the WCAG 2.2 tag are almost touching, and are using the same background colour.

Might it be best to select a different tag colour from grey, so that it doesn't get visually lumped in with the code snippets? Here's a list of the tag colours available with the new tag style: https://deploy-preview-2733--govuk-design-system-preview.netlify.app/components/tag/#additional-colours